.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Describe your changes
```

Comment:
project didn't have a PR template and so I added a paired down one that we use in other projects